lib/services/pages.test.js

```diff
@@ -26,6 +27,7 @@ describe(_.startCase(filename), function () {
     sandbox.stub(notifications, 'notify');
     sandbox.stub(timer);
     sandbox.stub(winston);
+    clock = sinon.useFakeTimers();
```

Comment:
Can use `sandbox.useFakeTimers();` so that it does not need to be manually restored.